pizzeria-backend/Models/Interfaces/OrderedProductsDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace pizzeria_backend.Models.Interfaces
+{
+    public class OrderedProductsDto
+    {
+        public int ProductId { get; set; }
+
+        public List<OrderedAddOnsDto>? AddOns { get; set; } = new List<OrderedAddOnsDto>();
```

Comment:
use IEnumerable interface instead of list

Reply:
Done